packages/screens/AdministrationDashboard/component/GuardiansBox.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { useState } from "react";
+import {
+  FlatList,
+  Image,
+  ImageBackground,
+  TouchableOpacity,
+  View,
+  useWindowDimensions,
+  ViewStyle,
+} from "react-native";
+
+import avaPNG from "../../../../assets/default-images/ava.png";
+import guardianPng from "../../../../assets/default-images/guardian_1.png";
+import addCircleSVG from "../../../../assets/icons/add-circle.svg";
+import checkBadgeSVG from "../../../../assets/icons/check-badge.svg";
+import dotSVG from "../../../../assets/icons/dot-more.svg";
+import downSVG from "../../../../assets/icons/down.svg";
+import trashSVG from "../../../../assets/icons/trash-white.svg";
+import upSVG from "../../../../assets/icons/up.svg";
+import { BrandText } from "../../../components/BrandText";
+import { SVG } from "../../../components/SVG";
+import { Box, BoxStyle } from "../../../components/boxes/Box";
+import { SearchInput } from "../../../components/sorts/SearchInput";
+import {
+  neutral33,
+  neutral00,
+  gradientColorLightBlue,
+} from "../../../utils/style/colors";
+import { fontSemibold14, fontSemibold16 } from "../../../utils/style/fonts";
+
+const MD_BREAKPOINT = 820;
+
+export const GuardiansBox = () => {
+  const { width } = useWindowDimensions();
+
+  const [guardiansHandler, setGuardiansHandler] = useState<number[]>([]);
+  const [isListOpen, setIsListOpen] = useState<boolean>();
+
+  return (
+    <View style={{ flexDirection: width >= MD_BREAKPOINT ? "row" : "column" }}>
+      {guardiansHandler?.map((d: any, index: any) => {
+        return (
+          <>
+            <View
+              style={[
+                viewBox,
+                { width: width >= MD_BREAKPOINT ? "19%" : "100%" },
+              ]}
+            >
+              <View style={insideBoxMap}>
+                <View style={dotBackground}>
+                  <BrandText style={[fontSemibold14]}>{index + 1}</BrandText>
+                </View>
+                <View>
+                  <SVG source={dotSVG} />
+                </View>
+              </View>
+              <Box notched style={herosLisBox}>
+                <TouchableOpacity
+                  onPress={() =>
+                    setIsListOpen((old: any) => (old === index ? null : index))
+                  }
+                  style={toggleBox}
+                >
+                  <BrandText style={[fontSemibold14]}>
+                    Genesis Guardians
+                  </BrandText>
+                  <View>
+                    {isListOpen === index ? (
+                      <SVG source={downSVG} />
+                    ) : (
+                      <SVG source={upSVG} />
+                    )}
+                  </View>
+                </TouchableOpacity>
+                {isListOpen === index ? (
```

Comment:
isListOpen type is boolean, index here is number so the comparison here is not correct